clash-prelude/src/Clash/Tutorial.hs

```diff
@@ -407,7 +439,13 @@ examine the 'register' function by taking a look at the first 4 samples of the
 [0,0,8,8]
 
 Where we see that the initial value of the signal is the specified 0 value,
-followed by 8's.
+followed by 8's. You might be surprised to see /two/ zeros instead of just a
+single zero. What happens is that Clash emulates what happens /before/ the
```

Comment:
I'm not a big fan of the word 'emulate'.

Perhaps: "What happens is that in Clash you get to see the output of the circuit /before/ the clock becomes actives. In other words, in Clash you get to describe the powerup values of registers too."